extract_repo_info.py

```diff
@@ -143,7 +143,7 @@ def _sleep_duration(failed_response: requests.Response) -> int:
     )
 
     if rate_limit_remaining < MIN_RATE_LIMIT_REMAINING:
-        sleep_time = rate_limit_reset_date - now
+        sleep_time = rate_limit_reset_date - now if rate_limit_reset_date > now else 1
```

Comment:
This should work. Another alternative is...

```suggestion
        sleep_time = max(1, rate_limit_reset_date - now)
```